plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/elastic/ElasticAgentExtension.java

```diff
@@ -132,12 +133,26 @@ public String getStatusReport(String pluginId) {
             throw new UnsupportedOperationException(format("Plugin `%s` implements Elastic Agent V1, `StatusReport` endpoint is not supported in this version.", pluginId));
```

Comment:
I think we would need this since V1 does not support this endpoint, the expectation from plugins implementing V2 is to support `get-capabilities` request